modules/phase_field/include/materials/EBCoupledVarTest.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
```

Comment:
Please add the moose modules copyright header and move test objects into the phase_field/test directory tree.